media/doc/README.WINE

```diff
@@ -289,7 +289,8 @@ kernel32 -
   reactos/dll/win32/kernel32/winnls/string/sortkey.c     # Synced to WineStaging-3.3
 
 lpk -
-  reactos/dll/win32/lpk/bidi.c                           # Synced to WineStaging-3.7
+  reactos/dll/win32/lpk/bidi.c                           # Synced to Wine-4.9
```

Comment:
Is there a reason not to sync to WineStaging anymore?

Reply:
Hmm? This got taken directly from the primary Wine repository (wine-devel?), IIRC WineStaging is a different repository.

Reply:
@SergeGautherie : because there are no LPK patches in the WineStaging repo (and no patch whatsoever for any bidi.c file), since LPK does not exist in Wine, and that this particular bidi.c file comes actually from usp10.c , for which there are no patches for in WineStaging.

Reply:
> This got taken directly from the primary Wine repository

Cherry-picking commits from Wine is fine, though that is somewhat different from doing an "upgrade" sync'.
My question was on the very README.WINE part.

> WineStaging is a different repository.

Yes, it is additional patches.

--

> because [...]

I'll skip the technical details. Then, you say "Synced to WineStaging-3.7" was actually a copy-pasta?